client/header/activity-panel/panels/inbox/card.js

```diff
@@ -1,25 +1,35 @@
 /**
  * External dependencies
  */
-import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { cloneElement, Component, Fragment } from '@wordpress/element';
+import { Button, Dropdown, Modal } from '@wordpress/components';
 import PropTypes from 'prop-types';
 import Gridicon from 'gridicons';
```

Comment:
I think this is unused in here.

Reply:
Yep, I don't see `<Gridicon />` in use anymore

Reply:
Thank you, I removed the Gridicon in the commit [7b8c37c](https://github.com/woocommerce/woocommerce-admin/pull/4256/commits/7b8c37c4729bbbb322fa63de4773ccef2f11c06d)